<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login_in</class>
 <widget class="QWidget" name="Login_in">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>311</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>账号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="username_line">
        <property name="styleSheet">
         <string notr="true">
background-color: rgb(247, 247, 247);
border:1px groove gray;border-radius:
7px;padding:2px 4px;
font: 10pt &quot;Candara&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>密码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password_line">
        <property name="styleSheet">
         <string notr="true">
background-color: rgb(247, 247, 247);
border:1px groove gray;border-radius:
7px;padding:2px 4px;
font: 10pt &quot;Candara&quot;;</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="login_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 170, 255,99);</string>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="regist_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 170, 255,99);</string>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>欢迎来到骶能家居系统</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
